app/decorators/node_decorator.rb

```diff
@@ -40,7 +40,11 @@ def list_graph_path
   end
 
   def tag_graph_path(tag_list)
-    h.tag_graph_path(tag_list, graph_parameter_list_params)
```

Comment:
I found an error occurs when `tag_list` is an empty string, this patch fixes it.